.nvmrc

```diff
@@ -1 +1 @@
-6.9
+11.10.1
```

Comment:
Is there a particular reason to go with node 11? Since October, 12 is LTS.

Reply:
Nope, that was the version I could get node-sass to compile with no problems! I just updated this to use 12.13.0, and node-was compiles for me fine. Could you delete your node_modules and try to `yarn install` again please?